contracts/src/mocks/MockWrappedNativeToken.sol

```diff
@@ -5,15 +5,15 @@
 
 pragma solidity 0.8.18;
 
-import {IWrappedNativeToken} from "./interfaces/IWrappedNativeToken.sol";
+import {IWrappedNativeToken} from "../interfaces/IWrappedNativeToken.sol";
 import {ERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/ERC20.sol";
 import {Address} from "@openzeppelin/contracts@4.8.1/utils/Address.sol";
 
 /**
  * THIS IS AN EXAMPLE CONTRACT THAT USES UN-AUDITED CODE.
  * DO NOT USE THIS CODE IN PRODUCTION.
  */
-contract WrappedNativeToken is IWrappedNativeToken, ERC20 {
+contract MockWrappedNativeToken is IWrappedNativeToken, ERC20 {
```

Comment:
I think you need to update `abi_bindings.sh` with the new file name and delete the old abi binding

Reply:
yup nice catch, I'm going to wati for OZ to confirm whether `WrappedNativeToken` was included in audit before proceeding with changes

Reply:
`WrappedNativeToken` was confirmed by OZ as audited in scope, updated report